src/ParametricUtils.jl

```diff
@@ -509,14 +509,8 @@ function solveGraphParametric(
   computeCovariance::Bool = true,
   solveKey::Symbol = :parametric,
   autodiff = :forward,
-  algorithm = Optim.NewtonTrustRegion, # Optim.BFGS,
```

Comment:
@dehann, just use the `algorithm` and `algorithmkwargs` keyword arguments if you want to use a different algorithm.

Reply:
alright, thanks.  I was testing stuff here.  Didn't realize you had explored the different algorithms to that extent yet.  Thanks for the fix